Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Modify-built-in-document-properties/Data/WordService.cs.bak

```diff
@@ -0,0 +1,32 @@
+﻿using Syncfusion.DocIO;
```

Comment:
Remove this unwanted file WordService.cs.bak

Reply:
Addressed the requested changes